homeassistant/components/dio_chacon/entity.py

```diff
@@ -0,0 +1,53 @@
+"""Base entity for the Dio Chacon entity."""
+
+from dio_chacon_wifi_api import DIOChaconAPIClient
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, MANUFACTURER
+from .coordinator import DioChaconDataUpdateCoordinator
+
+
+class DioChaconEntity(CoordinatorEntity):
```

Comment:
```suggestion
class DioChaconEntity(CoordinatorEntity[..]):
```
Please add the coordinator type here